### DISCUSSION TITLE
Feedback for "Better suggested pull request description from commit message"

### DISCUSSION THREAD

**Discussion by mfn** [community/community#33135] [Pull Requests]
Referring to https://github.blog/changelog/2022-09-07-better-suggested-pull-request-description-from-commit-message/

One, to me negative, side effect is that now when I write this in commit messages:

```
- foo
  bar
  baz
```
- foo
  bar
  baz

which renders as
![image](https://user-images.githubusercontent.com/87493/190322800-b2c664b1-1cff-44fe-98fc-7285be3da11c.png)
(i.e., it's aligned)

it comes out as
- foo bar baz

Basically all my nicely aligned bullet points having more than one line are broken?

Is there a way to have github simply keep the message 1:1 the way it is in the git commit message? This causes me now more work than before to have the message nicely rendered. Basically, before I had 0 work to do, now I need to think about/fix the formatting?

thanks
